tracardi/service/storage/drivers/elastic/validation_schema.py

```diff
@@ -0,0 +1,20 @@
+from tracardi.service.storage.factory import storage_manager
+from tracardi.domain.event_payload_validator import PayloadValidatorRecord, EventPayloadValidator
+
+
+async def add_schema(schema: EventPayloadValidator):
+    schema = schema.encode().dict()
+    return await storage_manager("validation-schema").upsert(schema)
+
+
+async def del_schema(event_type: str):
+    return await storage_manager("validation-schema").delete(event_type)
+
+
+async def load_schemas(start: int = 0, limit: int = 10):
+    return await storage_manager("validation-schema").load_all(start, limit)
+
+
+async def get_schema(event_type: str) -> EventPayloadValidator:
```

Comment:
It should rather throw an exception than return the result. We can not do anything with the result.